src/diffusers/pipelines/deprecated/__init__.py

```diff
@@ -0,0 +1,153 @@
+from typing import TYPE_CHECKING
```

Comment:
Can you add a short README.md to this folder that says that we won't accept any changes to files in this folder? E.g. just:

```
# Deprecated Pipelines

This folder contains pipelines that have very low usage as measured by model downloads, issues and PRs. While you can still use the pipelines just as before, we will stop testing the pipelines and will not accept any changes to existing files.